src/libs/Navigation/linkingConfig/config.ts

```diff
@@ -298,6 +298,12 @@ const config: LinkingOptions<RootStackParamList>['config'] = {
                         [SCREENS.WORKSPACE.TAG_CREATE]: {
                             path: ROUTES.WORKSPACE_TAG_CREATE.route,
                         },
+                        [SCREENS.WORKSPACE.TAG_SETTINGS]: {
+                            path: ROUTES.WORKSPACE_TAG_SETTINGS.route,
+                            parse: {
+                                tagName: (tagName: string) => encodeURIComponent(tagName),
```

Comment:
We need to decode the route here
```suggestion
                                tagName: (tagName: string) => decodeURIComponent(tagName),
```